packages/atomic/src/pages/v1.html

```diff
@@ -201,6 +201,7 @@
         <atomic-rating-facet field="snrating" label="Rating" number-of-intervals="5"> </atomic-rating-facet>
         <atomic-rating-range-facet field="snrating" label="Rating Range" number-of-intervals="5">
         </atomic-rating-range-facet>
+        <atomic-color-facet field="author" label="Colors"></atomic-color-facet>
```

Comment:
I could not find a field in the Search UI Samples org that had color names.

Reply:
maybe using something like `source` for a field, Source as label, the value set is smaller and color mapping source type makes more sense than authors. It's better than nothing!

Reply:
Or maybe we can leverage some of the Icons we have for the `filetype` field they are in the `assets` folder

Reply:
Bump